src/nile/cli.py

```diff
@@ -234,9 +237,13 @@ def debug(tx_hash, network, contracts_file):
 
 @cli.command()
 @network_option
-def get_accounts(network):
+@click.option("--predeployed/--registered", default=False)
```

Comment:
I was checking Click, and as you can check [here](https://click.palletsprojects.com/en/8.1.x/options/#boolean-flags), Click always wants you to provide an enable and disable flag so that you can change the default later, but just for this, the disable flag is not necessary and is not required from CLI, so I thought that with documenting the --predeployed is enough, because is never needed to pass the --registered flag. Let me know what you think.

Reply:
Yeah, that's reasonable. Documenting `--predeployed` seems like enough